03.bowling/bowling.rb

```diff
@@ -12,26 +12,36 @@
   end
 end
 
-frames = []
-shots.each_slice(2) do |s|
-  frames << s
-end
+frames = shots.each_slice(2).to_a
 
-point = 0
-frames.slice(0..9).each_with_index do |frame, i|
-  point += if i == 9 && frames[i + 2]
-             frame.sum + frames[i + 1][0] + frames[i + 2][0]
-           elsif i == 9 && frames[i + 1]
-             frame.sum + frames[i + 1].sum
-           elsif i < 9 && frame[0] == 10 && frames[i + 1][0] == 10 && frames[i + 2][0]
-             frame.sum + frames[i + 1][0] + frames[i + 2][0]
-           elsif i < 9 && frame[0] == 10 && frames[i + 1]
-             frame.sum + frames[i + 1][0..1].sum
-           elsif i < 9 && frame.sum == 10 && frames[i + 1]
-             frame.sum + frames[i + 1][0]
-           else
-             frame.sum
-           end
+point = frames [0..9].each_with_index.sum do |frame, i|
+  next_frame = frames[i + 1]
+  next_2_frame = frames[i + 2]
+  point = if i == 9
```

Comment:
```suggestion
  if i == 9
```

ブロックの戻り値が加算されてsumメソッドの戻り値になるので変数代入は不要です

Reply:
修正済みです！